core/src/test/java/dk/alexandra/fresco/lib/compare/bool/eq/TestBinaryEquality.java

```diff
@@ -0,0 +1,27 @@
+package dk.alexandra.fresco.lib.compare.bool.eq;
+
+import java.util.ArrayList;
+import java.util.List;
+import dk.alexandra.fresco.framework.value.SBool;
+import dk.alexandra.fresco.framework.DRes;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+
+public class TestBinaryEquality {
```

Comment:
Class name could probably be a bit better. What about TestBinaryEQInputLength? The other name indicates a different set of tests.

Reply:
I don't agree. I believe the class name should reflect the class being tested and then the method names can indicate exactly what is being tested. This is the naming model we've used for the other tests as well, eg. "TestBitSetUtils", "TestCreditRater", etc.

Reply:
I agree with Michael. This way you can extend the test class with more tests on the same class without having to change the name.

Reply:
Okay, makes sense. Ignore comment then.